website/server/models/user/schema.js

```diff
@@ -521,6 +521,7 @@ let schema = new Schema({
     return {};
   }},
   pushDevices: [PushDeviceSchema],
+  ABtest: {type: String},
```

Comment:
It's the way private properties work, they don't show up in the API and so
in integration tests. I'd say either move it to the pre save hook and use a
unit test or just make the property public, it doesn't contain sensitive
data anyway. It was nice to have but not necessary

Il 08 set 2016 23:52, "Sabe Jones" <notifications@github.com> ha scritto:

> In website/server/models/user/schema.js
> <https://github.com/HabitRPG/habitrpg/pull/7984#discussion_r78096265>:
>
> > @@ -521,6 +521,7 @@ let schema = new Schema({
> >      return {};
> >    }},
> >    pushDevices: [PushDeviceSchema],
> > +  ABtest: {type: String},
>
> Shoot, that makes it show up undefined during the integration test. Is
> the test wrong, or the way I found to make the property private?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/HabitRPG/habitrpg/pull/7984/files/9efa85423bf134e9708c11b30ac4264a66287675#r78096265>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAkAV1tDbnw_sgG9WogSyHKAdLm7UQ6Nks5qoIOtgaJpZM4J4b0S>
> .
>